prow/git/git.go

```diff
@@ -310,22 +314,33 @@ func (r *Repo) CheckoutPullRequest(number int) error {
 // Config runs git config.
 func (r *Repo) Config(key, value string) error {
 	r.logger.Infof("Running git config %s %s", key, value)
-	if b, err := r.gitCommand("config", key, value).CombinedOutput(); err != nil {
+	if b, err := r.gitCommand("config", key, value); err != nil {
 		return fmt.Errorf("git config %s %s failed: %v. output: %s", key, value, err, string(b))
 	}
 	return nil
 }
 
+// Remote builds a remote url from user, pasword, and a slice of path items
+func Remote(base *url.URL, user string, pass string, pathItems ...string) *url.URL {
+	copy := *base
+	if user != "" && pass != "" {
+		copy.User = url.UserPassword(user, pass)
+	}
+	if len(pathItems) != 0 {
```

Comment:
Why does this need to be conditional?

Reply:
Idiomatic Go is to have as few levels of indentation as possible, so let's omit this.